src/main/resources/avro/bdg.avdl

```diff
@@ -411,34 +449,36 @@ record StructuralVariant {
 record Variant {
 
   /**
-   The Phred scaled error probability of a variant, given the probabilities of
-   the variant in a population.
-   */
-  union { null, int } variantErrorProbability = null;
-
-  /**
-   The reference contig that this variant exists on.
+   The reference contig that this variant exists on.  VCF column 1 "CONTIG".
    */
   union { null, string } contigName = null;
 
   /**
    The 0-based start position of this variant on the reference contig.
+   VCF column 2 "POS" converted to zero-based coordinate system, closed-open intervals.
    */
   union { null, long } start = null;
 
   /**
    The 0-based, exclusive end position of this variant on the reference contig.
+   Calculated by start + referenceAllele.length().
    */
   union { null, long } end = null;
 
   /**
-   A string describing the reference allele at this site.
+   Zero or more of unique names or identifiers for this variant. If this is a dbSNP variant it is
+   encouraged to use the rs number(s).  VCF column 3 "ID".
+   */
+  array<string> names = [];
```

Comment:
GA4GH use a strong definition of `id`
https://github.com/ga4gh/schemas/blob/master/doc/source/api/apidesign_intro.rst#object-ids
https://github.com/ga4gh/schemas/blob/master/doc/source/api/apidesign_intro.rst#id-and-name

I don't believe we care.  What are you proposing?